app/src/main/res/values/strings.xml

```diff
@@ -97,4 +97,22 @@
   <string name="add_profile_error_name_only_letters">Names can only have letters. Try another name?</string>
   <string name="add_profile_error_pin_length">Your PIN should be 3 digits long.</string>
   <string name="add_profile_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
+  <!-- PinPasswordActivity. -->
+  <string name="pin_password_forgot_pin">I FORGOT MY PIN</string>
```

Comment:
This should be `I forgot my pin`, normally from xml code we should make this allcaps.

Reply:
Ditto elsewhere in this PR.

Reply:
I updated all that I could. I didn't update the strings used in dialogs since I can't access their xml.